npm-debug.log

```diff
@@ -0,0 +1,1091 @@
+0 info it worked if it ends with ok
```

Comment:
This file shouldn't be committed. Do you mind adding this to `.gitignore`: `npm-debug.log*`, and deleting this file?